python/sdk/prelude_sdk/controllers/partner_controller.py

```diff
@@ -33,10 +33,24 @@ def detach(self, partner_code: int):
         if res.status_code == 200:
             return res.json()
         raise Exception(res.text)
+
+    @verify_credentials
+    def block(self, partner_code: int, test_id: str):
+        """ Report to a partner to block a test """
+        params = dict(test_id=test_id)
+        res = requests.post(
+            f'{self.account.hq}/partner/block/{partner_code}',
+            headers=self.account.headers,
+            json=params,
+            timeout=30
+        )
+        if res.status_code == 200:
+            return res.json()
```

Comment:
also what does this return?

Reply:
@stephanwampouille we'll want this to go on the JS-SDK as well.